templates/master/01-master-kubelet/_base/units/kubelet.yaml

```diff
@@ -3,7 +3,7 @@ enabled: true
 contents: |
   [Unit]
   Description=Kubernetes Kubelet
-  Wants=rpc-statd.service
+  Wants=rpc-statd.service crio.service
```

Comment:
That's not going to guarantee what you want though - the crio socket will still be created async right?  I think we want both `Requires=crio.service` and `After=crio.service`.

Reply:
(lots more detail in `man systemd.unit`)